src/source/authentication.rst

```diff
@@ -195,6 +201,8 @@ specified in the request at step 1 with the following GET parameters
     GET /code?code=AUTH_CODE&state=YOUR_STATE
     Host: yoursite.com
 
+This is how your server gets the short-lived authorization code to subsequently get a refresh & access token.
```

Comment:
It's not really "your server" but more "your client".

Reply:
spot on